proof.go

```diff
@@ -106,6 +106,11 @@ func NewAbsenceProof(proofStart, proofEnd int, proofNodes [][]byte, leafHash []b
 	return Proof{proofStart, proofEnd, proofNodes, leafHash, ignoreMaxNamespace}
 }
 
+// IsOfEmptyProof checks whether the proof corresponds to an empty proof as defined in NMT specifications https://github.com/celestiaorg/nmt/blob/master/docs/spec/nmt.md.
+func (proof Proof) IsOfEmptyProof() bool {
```

Comment:
```suggestion
// IsEmptyProof checks whether the proof corresponds to an empty proof as defined in NMT specifications https://github.com/celestiaorg/nmt/blob/master/docs/spec/nmt.md.
func (proof Proof) IsEmptyProof() bool {
```
nit

Reply:
Thank you for your suggestion. The reason behind this naming is to maintain consistency with the [IsOfAbsence()](https://github.com/celestiaorg/nmt/blob/0c70c1bde94bf2686e854a072715eaf361ac59ef/proof.go#L68) function, that implements a similar logic.

Reply:
Oh, I see your point, renamed it in https://github.com/celestiaorg/nmt/pull/180/commits/9e3d8de24adb44c98c3ca644b56da63e1f38a715